posts/git-cherry-pick/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Cherry-Pick on GitHub"
+author: "Robyn Marowitz"
+date: "2023-09-26"
+description: "Committing a mistake in GitHub can feel so scary, cherry picking is a tool to help you fix your history when something like that happens. This post lays out simple and clear instructions on the cherry picking method."
```

Comment:
```suggestion
description: "Committing a mistake in GitHub can feel so scary. Cherry picking is a tool to help you fix your history when something like that happens. This post lays out instructions on the cherry picking method."
```

Or maybe `!` instead of `.`.... or :scream: emoji ;) 

I think "simple and clear" could be removed. Words like "easy", "simple", "_just_ do X" can trivialize the experience of someone who is struggling to learn a new concept.

"Committing a mistake in GitHub" is a bit unclear to me; commits happen in Git and you push them to GitHub. I suppose some might edit and commit directly in GitHub, but I think the more common workflow is using `git` or an editor that integrates with Git. Maybe could be worded as "Pushing a mistake to GitHub" or "Committing a mistake in Git"?